docs/src/components/Navbar/Links.tsx

```diff
@@ -32,7 +32,7 @@ const LinkIcon = ({ name }: { name?: string }) => {
     case "Accessibility":
       return <IrisScanIcon {...iconStyles} />;
     case "Playground":
-      return <Code {...iconStyles} />;
+      return <Code size="medium" />;
```

Comment:
This size applies the same 20px size and the `stroke` prop is actually the default color value for an Icon. This was not marked as type error because props were not typed, but it was working properly because the size and colors used are the default ones 😅

Reply:
Good think we added types then :)